docs/examples/metrics/plot_lcss.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""
+LCSS computation
```

Comment:
```suggestion
Longest Common Subsequence
```

Since there is this possible misunderstanding with longest common substring, I would prefer that the title be explicit about which algorithm we have in store.